_aggregations/metric/matrix-stats.md

```diff
@@ -27,60 +42,78 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
+      "doc_count": 4675,
+      "fields": [
+        {
+          "name": "products.base_price",
+          "count": 4675,
+          "mean": 34.99423943014724,
+          "variance": 360.5035285833702,
+          "skewness": 5.530161335032689,
+          "kurtosis": 131.1630632404217,
+          "covariance": {
+            "products.base_price": 360.5035285833702,
+            "taxful_total_price": 846.6489362233169
+          },
+          "correlation": {
+            "products.base_price": 1,
+            "taxful_total_price": 0.8444765264325269
+          }
         },
-        "correlation" : {
-          "products.base_price" : 0.8444765264325268,
-          "taxful_total_price" : 1.0
+        {
+          "name": "taxful_total_price",
+          "count": 4675,
+          "mean": 75.05542864304839,
+          "variance": 2788.1879749835425,
+          "skewness": 15.812149139923994,
+          "kurtosis": 619.1235507385886,
+          "covariance": {
+            "products.base_price": 846.6489362233169,
+            "taxful_total_price": 2788.1879749835425
+          },
+          "correlation": {
+            "products.base_price": 0.8444765264325269,
+            "taxful_total_price": 1
+          }
         }
-      }
-    ]
+      ]
+    }
   }
- }
 }
 ```
 
-The following table lists all response fields.
+The following table describes the response fields:
 
-Statistic | Description
-:--- | :---
-`count` | The number of samples measured.
-`mean` | The average value of the field measured from the sample.
-`variance` | How far the values of the field measured are spread out from its mean value. The larger the variance, the more it's spread from its mean value.
-`skewness` | An asymmetric measure of the distribution of the field's values around the mean.
-`kurtosis` | A measure of the tail heaviness of a distribution. As the tail becomes lighter, kurtosis decreases. As the tail becomes heavier, kurtosis increases. To learn about kurtosis, see [Wikipedia](https://en.wikipedia.org/wiki/Kurtosis).
-`covariance` | A measure of the joint variability between two fields. A positive value means their values move in the same direction and the other way around.
-`correlation` | A measure of the strength of the relationship between two fields. The valid values are between [-1, 1]. A value of -1 means that the value is negatively correlated and a value of 1 means that it's positively correlated. A value of 0 means that there's no identifiable relationship between them.
\ No newline at end of file
+| Statistic    | Description |
+| :---         | :---        |
+| `count`      | The number of documents sampled for the aggregation. |
+| `mean`       | The average value of the field computed from the sample. |
+| `variance`   | The square of deviation from the mean, a measure of data spread. |
+| [`skewness`](https://en.wikipedia.org/wiki/Skewness)   | A measure of the distribution's assymetry about the mean. |
```

Comment:
Not sure if these stat names themselves should be clickable? I liked previous mode of saying, to learn more about xxxx see....

Reply:
Style guide specifies direct links in tables to save space.